src/linkplay/utils.py

```diff
@@ -71,7 +73,18 @@ def decode_hexstr(hexstr: str) -> str:
 
 def create_unverified_context() -> ssl.SSLContext:
     """Creates an unverified SSL context."""
-    sslcontext: ssl.SSLContext = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+    dirs = AppDirs("python-linkplay")
+    mtls_certificate_path = os.path.join(dirs.user_data_dir, "linkplay.pem")
+
+    if not os.path.isdir(dirs.user_data_dir):
+        os.mkdir(dirs.user_data_dir)
```

Comment:
```suggestion
        os.makedirs(dirs.user_data_dir, exist_ok=True)
```

`os.mkdir` does not create parent dirs